packages/drivers/local-driver/src/localDocumentStorageService.ts

```diff
@@ -82,6 +90,9 @@ export class LocalDocumentStorageService implements IDocumentStorageService {
 		summary: ISummaryTree,
 		context: ISummaryContext,
 	): Promise<string> {
+		if (context.referenceSequenceNumber === 0) {
+			await createDocument(this.localDeltaConnectionServer, this.resolvedUrl, summary);
```

Comment:
should this return? seems a bit weird to create with a summary, and then upload that same summary

Reply:
What should I return? Currently it's redundant but I don't see changing createDocument return type as a viable option.

Reply:
you should call get versions, which i believe will return the handle of the summary you just uploaded.